cocos/core/settings.ts

```diff
@@ -74,6 +74,10 @@ export class Settings {
             }
         }
         if (!path) return Promise.resolve();
+        if (window.oh) {
+            this._settings = require('../settings.json');
+            return Promise.resolve();
```

Comment:
Why does OH need to hack this?

Reply:
OpenHarmony temporarily does not support reading json that is not in the resource directory

Reply:
Then it is better to add TODO. And remove TODO after it supports it.